packages/integration-tests/index.js

```diff
@@ -359,8 +360,8 @@ async function addCollections(stackName, bucketName, dataDirectory, postfix,
       collection.duplicateHandling = duplicateHandling;
     }
     const c = new Collection();
-    console.log(`\nadding collection ${collection.name}___${collection.version}`);
-    return c.delete({ name: collection.name, version: collection.version })
+    console.log(`Adding collection ${collection.name}___${collection.version}`);
```

Comment:
Would it be better to use the `constructCollectionId` helper here so that we don't have to hardcode the string format?

Reply:
I thought about that, but it would require another import, and this is only using it for logging, but I'd be happy to pull it in, if you think it's worth it.

Reply:
Hmm yeah. I guess I would probably pull it in to avoid the inconsistencies in the future